client/extensions/woocommerce/app/order/order-details/table.js

```diff
@@ -284,9 +300,12 @@ class OrderDetailsTable extends Component {
 					<OrderTotalRow
 						currency={ order.currency }
 						label={ translate( 'Shipping' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Shipping address' )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/store-order-edit-shipping)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).